universal-login-relayer/src/core/services/execution/deployment/DeploymentHandler.ts

```diff
@@ -22,7 +24,7 @@ class DeploymentHandler {
       deviceInfo,
       state: 'Queued',
       contractAddress,
-      refundPayerId: apiKey && (await this.refundPayerValidator.store.get(apiKey))?.id,
+      refundPayerId: apiKey && (await this.refundPayerStore.get(apiKey))?.id,
```

Comment:
And now refundPayerStore can be private in RefundPayerValidator 😄

Reply:
Yeap, i forgot about it :D